feature-libs/product-configurator/yarn.lock

```diff
@@ -0,0 +1,3694 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```

Comment:
I'm not sure we should have yarn.lock in /product-configurator dir...

Reply:
Removed